src/main/java/org/jason/fgcontrol/flight/position/KnownRoutes.java

```diff
@@ -19,6 +19,18 @@ private KnownRoutes() {}
 
     private final static Logger LOGGER = LoggerFactory.getLogger(KnownRoutes.class);
 
+    public final static ArrayList<WaypointPosition> NA_CALIFORNIA = new ArrayList<WaypointPosition>() {
+        // Starting position for the route: Stanford -> SFO -> SANJOSE -> SACRAMENTO -> TAHOE -> Stanford with starting position of Standford University location 37.4276642,-122.1726403
```

Comment:
Add the start heading in this comment too. From Stanford to SFO it looks like the heading is 328.11.

Reply:
updated comments with initial heading